benchmark/clblast/cli_device_selector.hpp

```diff
@@ -38,65 +38,34 @@
 #include <utility>
 #include <vector>
 
+#include "cli_args.hpp"
+
 class cli_device_selector {
   std::string program_name;
 
-  void usage() {
```

Comment:
Where has the usage gone? I'm assuming that it's now part of the command line arg parsing code?

Reply:
Yes, it is handled by the arg parser